scripts/cybershake/add_to_mgmt_queue.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+"""Wrapper script used by the templates to add updates to the mgmt db queue"""
+import argparse
+
+from shared_workflow import shared
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument("queue_folder", type=str, help="Mgmt db queue folder")
+    parser.add_argument("run_name", type=str, help="The realisation/run name")
+    parser.add_argument(
+        "proc_type", type=int, help="The integer value for the process type"
+    )
+    parser.add_argument("status", type=int, help="The integer value for the status")
```

Comment:
Any reason you used integer values rather than strings? 
I feel like integer values expose too much of the magic behind the curtain.

Reply:
This isn't really a user facing script, its only used from within the slurm scripts.

Reply:
The magic numbers for this in the slurm scripts isn't the best either

Reply:
I think that if there are changes to the IDs. Say to avoid giving pre-processing tasks negative numbers. Then it's not obvious (from a search) that this code has to also change. If we are explicit with str names, it should be much more obvious when something 'wrong' is happening.

Reply:
Hmh I don't think we should ever change the IDs, since they are IDs and don't need to be in order. I prefer magic number to magic string, but I'll change it to strings.

Reply:
i would personally prefer strings as well, this will make debug or any manual override a lot easier.